pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -377,6 +381,23 @@ func podSpec(containers []corev1.Container, opts ...podSpecOption) *corev1.PodSp
 	return podSpec
 }
 
+func withAppendedTokenVolumes(filenames []string, audiences []string, expires []*int64) podSpecOption {
```

Comment:
It feels like this would be cleaner as a list-of-struct rather than parallel arrays. This is a test, so slightly less worried.

Another option would be to explicitly take advantage of e.g. `expires` being the same for all the tokens.

Reply:
Similarly, since we know we want `expires` to be non-`nil`, why not have `[]int64` if we need it, and do the extract-to-pointer in the function?

Reply:
Changed to list-of-struct

Reply:
Changed to int64 and extract-to-pointer in the function